<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gp_gestaoestoque</class>
 <widget class="QDialog" name="gp_gestaoestoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>711</width>
     <height>411</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_novoproduto">
    <attribute name="title">
     <string>Novo Produto</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>662</width>
       <height>246</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Código do Produto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_codigo"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Descrição do Produto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_descricao"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Fornecedor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_fornecedor"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Qtde. Estoque</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_qtdestoque"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Valor de Compra</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_valorcompra"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Valor de Venda</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_valorvenda"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_novoproduto">
          <property name="text">
           <string>Novo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_gravarnovoproduto">
          <property name="text">
           <string>Gravar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_gestaoprodutos">
    <attribute name="title">
     <string>Gestão Produtos</string>
    </attribute>
    <widget class="QTableWidget" name="tw_g_produtos">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>101</y>
       <width>371</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Filtrar por:</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>122</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QRadioButton" name="rb_g_codigo">
         <property name="text">
          <string>Cód. Produto</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_g_produto">
         <property name="text">
          <string>Descrição Produto</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>281</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_16">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Código do Produto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_g_codigoproduto"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_11">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Produto</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txt_g_produto"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Fornecedor</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txt_g_fornecedor"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="labelestoque">
              <property name="text">
               <string>Quantidade</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_g_qtdestoque"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Valor Compra</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_g_valorcompra"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Valor Venda</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_g_valorvenda"/>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btn_g_gravar">
          <property name="text">
           <string>Gravar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_g_excluir">
          <property name="text">
           <string>Excluir</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>10</y>
       <width>211</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_18">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_17">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Filtrar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_g_filtrar"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_g_filtrar">
        <property name="text">
         <string>Pesquisar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
